activerecord/test/cases/adapters/postgresql/quoting_test.rb

```diff
@@ -34,6 +34,42 @@ def test_quote_range
           assert_equal "'[1,0]'", @conn.quote(type.serialize(range))
         end
 
+        def test_quote_range_date
+          range = (Date.parse("2010-01-01")...Date.parse("2010-01-01"))
+          type = OID::Range.new(Type::DateTime.new, :tsrange)
+          assert_equal "'[''2010-01-01'',''2010-01-01'')'", @conn.quote(type.serialize(range))
```

Comment:
The round trip tests are sufficient, I don't think we need to test this quoting behavior.

Reply:
Gotcha, so remove *all* the tests introduced in this commit?

Reply:
Just the ones in this file. The ones in `range_test.rb` all look good